source/controller/project_controller.py

```diff
@@ -220,6 +221,7 @@ def on_about(self):
 
     def on_restore(self):
         self.project_window.restoreState(self.settings.value("state"))
+        self.on_visualize_tree()
```

Comment:
It is necessary refactor the `self.settings.setValue`, because this is also causing the bug of restarting the project after closing the wizard. The status saving is being done by recreating all docks from scratch, which is wrong.